frontend/src/containers/Results/Results.js

```diff
@@ -11,9 +11,9 @@ class Results extends React.Component {
     requests: [],
     page: 1,
     order: 'desc',
-  }
+  };
 
-  componentWillMount() {
+  UNSAFE_componentWillMount() {
```

Comment:
not sure on how to solve these ones for placing order, since the bot now won't recognize they are lifecycle methods and the ordering shouldn't change.
Maybe it's worth actually removing these methods and refactor a bit the components according to what React recommends now. 

wdyt?